### DISCUSSION TITLE
How does dpdata convert lammps traj.lammpstrj to xyz format

### DISCUSSION THREAD

**Discussion by twoslucky** [deepmodeling/dpdata#593] [General]
### Summary

How does dpdata convert lammps traj.lammpstrj to xyz format

### dpdata Version

dpdata

### Details

How does dpdata convert lammps traj.lammpstrj to xyz format

**Comment by wanghan-iapcm** [comment]
the dpdata does not support output in `quip/gap/xyz` format

**Comment by njzjz** [comment]
Pure xyz is supported: https://docs.deepmodeling.com/projects/dpdata/en/master/formats/XYZFormat.html
